velox/exec/tests/TableScanTest.cpp

```diff
@@ -49,6 +50,7 @@ using namespace facebook::velox::exec;
 using namespace facebook::velox::common::test;
 using namespace facebook::velox::exec::test;
 using namespace facebook::velox::tests::utils;
+using namespace facebook::velox::dwrf;
```

Comment:
This is causing build issues due to ambiguity. See the PR CI pipeline.

Reply:
Thank you! Removed that line and ran `make debug` and it was successful for me.